cheatsheets/Abuse_Case_Cheat_Sheet.md

```diff
@@ -113,15 +105,15 @@ It is important to take into account **Technical** and **Business** kind of abus
 *Example:*
 
 - Technical flagged abuse case: Add Cross Site Scripting injection into a comment input field.
-- Business flagged abuse case: Ability to modify  the price of an article in an online causing the user to pay a lower amount for the wanted article.
+- Business flagged abuse case: Ability to modify the price of an article in an online causing the user to pay a lower amount for the wanted article.
```

Comment:
Change to something like: `Ability do modify the price of an article causing the user to pay a lower amount for the wanted article`.
I removed `in an online`. Unless you can fit a better sentence for it